spec/identity/hostdata/ec2_spec.rb

```diff
@@ -4,21 +4,16 @@
   describe '.load' do
     subject(:load) { Identity::Hostdata::EC2.load }
 
-    let(:ec2_api_token) { SecureRandom.hex }
-
     it 'loads data from the magic ECS URL' do
-      stub_request(:put, 'http://169.254.169.254/latest/api/token').
-          with(headers: { 'X-Aws-Ec2-Metadata-Token-Ttl-Seconds' => '60' }).
-          to_return(body: ec2_api_token)
-      stub_request(:get, 'http://169.254.169.254/2016-09-02/dynamic/instance-identity/document').
-        with(headers: { 'X-aws-ec2-metadata-token' => ec2_api_token }).
-        to_return(body: document.to_json)
+      stub_ec2_metadata(document: document)
 
       ec2 = load
       expect(ec2.region).to eq('us-west-2')
     end
 
     it 'blows up when the request times out' do
+      ec2_api_token = SecureRandom.hex
+
       stub_request(:put, 'http://169.254.169.254/latest/api/token').
```

Comment:
I wanted to remove this pair of `stub_request` here, but I didn't find a way I liked
- Option with some weird control flow to stub a timeout
- Separate helper method with a timeout just used once
- Refactor out methods for each request stub, but they're always used together except this once

So I just went with having the specialized stubs here